index.src.html

```diff
@@ -2459,7 +2459,7 @@
 
   : <dfn>Relying Party</dfn>
   : <dfn>WebAuthn Relying Party</dfn>
-  :: The entity which needs to rely in the authentication provided by the WebAuthn specification. At the conclusion of the account creation, the Relying Party has the public key that was created by the Authenticator.
+  :: The entity which needs to rely in the authentication provided by the WebAuthn specification. At the conclusion of the registration, the Relying Party has the public key that was created by the Authenticator.
```

Comment:
"At the conclusion of the registration" sounds weird. Can we say "When registration concludes", or if you want to link to the section on this, "When [[#registration-embedded]] concludes"?